src/OnboardingSPA/components/LivePreview/BlockPreview/index.js

```diff
@@ -25,9 +25,18 @@ const BlockPreview = ( {
 	viewportWidth = 1300,
 	styling = 'large',
 	previewSettings = false,
+	animationDuration = 2500,
 } ) => {
 	const [ blocks, setBlocks ] = useState();
 	const [ settings, setSettings ] = useState();
+	const [ isShown, setIsShown ] = useState(false);
+
+	useEffect(() => {
+		const timer = setTimeout(() => {
+			setIsShown(true);
+		}, animationDuration);
+		return () => clearTimeout(timer);
+	}, [animationDuration]);
```

Comment:
```suggestion
	useEffect(() => {
	setTimeout(() => {
			setIsShown(true);
		}, animationDuration);
	}, []);
```

Reply:
Basically, `return () => clearTimeout(timer);` is the cleanup code useEffect is something that will run on every render and due to which the clearTimeout is something that makes sure the cleanup code runs and the Timeout doesn't run everytime.